examples/echo-test/echo-test-template.cfg

```diff
@@ -0,0 +1,26 @@
+settings['DEFAULT'] = {
```

Comment:
The same, if you moves the echo-test to tools/, you could also move the echo-test-template.cfg as a `matrix-echo.cfg.example` like https://github.com/psaavedra/matrix-bot/blob/master/cfg/matrix-bot.cfg.example and add it in the setup.py file https://github.com/psaavedra/matrix-bot/blob/master/setup.py#L47